IoT/vss-extension.json

```diff
@@ -102,12 +101,12 @@
             "environments": [{
               "name": "dev",
               "deployedResourceIds": [
-                "/subscriptions/{{inputs.subscriptionId}}/resourceGroups/{{inputs.resourceGroup}}/providers/Microsoft.Compute/virtualMachines/{{inputs.vmName}}",
+                "/subscriptions/{{inputs.subscriptionId}}/resourceGroups/{{inputs.resourceGroup}}/providers/Microsoft.Compute/virtualMachines/{{{inputs.iotHubName}}}-edge-simulation",
                 "/subscriptions/{{{inputs.subscriptionId}}}/resourceGroups/{{{inputs.resourceGroup}}}/providers/Microsoft.Devices/IotHubs/{{{inputs.iotHubName}}}"
               ],
               "variables": {
                 "vmPassword": {
-                  "value": "{{{inputs.vmPassword}}}",
+                  "value": "{{{system.projectName}}}abcDEF123^$*",
```

Comment:
what is this random string? Did {{#tinyguid}}{{/tinyguid}} not work for you?

Reply:
fixed